packages/core/useMouseInElement/demo.vue

```diff
@@ -19,11 +19,37 @@ const stringify = reactify(
 const target = useTemplateRef<HTMLElement>('target')
 const mouse = reactive(useMouseInElement(target))
 const text = stringify(mouse)
+
+const inlineTarget = useTemplateRef<HTMLElement>('inline-target')
+const inlineMouse = reactive(useMouseInElement(inlineTarget))
+const inlineText = stringify(inlineMouse)
 </script>
 
 <template>
   <div flex="~" gap="4">
     <Area ref="target" />
     <pre lang="yaml">{{ text }}</pre>
   </div>
+
+  <div flex="~" gap="4">
+    <div
+      class="el"
+      w="40"
+      h="40"
+      bg="gray-400/20"
+      border="rounded"
+      flex="~"
+      place="content-center"
+      select="none"
+      style="text-align: center"
+    >
+      <div m="auto">
+        <span>
+          Hover <span ref="inline-target" style="background: rgb(36, 146, 82, 0.5)">
```

Comment:
background color is the green color used for highlights on `vueuse` page, just lowered the opacity to not mask the actual text.